### ISSUE TITLE
Continues Tunnel Issue

### ISSUE THREAD

**Issue by BagiIT** [ST-Apps/CS2-ExtendedRoadUpgrades#28]
I dont know if this is inteded behaviour but when i upgrade a road to a tunnel it works just fine, but is the road has 2 or more segments aka. nodes the tunnel breaks as seen in the image. Gmae version is 1.0.15f1 and mod version is 1.3.0
![4](https://github.com/ST -Apps/CS2-ExtendedRoadUpgrades/assets/56765679/8d7e62d7-ad7a-4296-bdec-29b282029bc0)

**Comment by graup** [comment]
Yeah, I also noticed this. Upgrading multiple segments to a tunnel does not give the same result as drawing a new multi-segmented tunnel. I don't think that's a rendering issue, somehow there must be a difference between a "tunnel entrance" node and a "tunnel continuation" node.

**Comment by ST-Apps** [comment]
I initially misunderstood the issue, potentially fixed in #35
